src/rest.cpp

```diff
@@ -490,21 +490,22 @@ static bool rest_getutxos(HTTPRequest* req, const std::string& strURIPart)
     std::vector<bool> hits;
     bitmap.resize((vOutPoints.size() + 7) / 8);
     {
-        auto process_utxos = [&vOutPoints, &outs, &hits](const CCoinsView& view, const CTxMemPool& mempool) {
+        auto process_utxos = [&vOutPoints, &outs, &hits](const CCoinsView& view, const CTxMemPool& tx_pool) {
```

Comment:
I don't understand your insistence on using "transaction pool" in comments, function and variables names. Adding another name for something that is already commonly called mempool just adds to confusion for new contributors. Can we just call the mempool "the mempool" and be done with it?

Reply:
tend to agree. In fact, there are more "transaction pools" like "orphan pool" etc. "Mempool" seems unambiguous in comparison?

Reply:
I'd rather not use `mempool` here because it shadows the global of the same name. This is fragile, because the code could be moved around and change its meaning because instead of `process_utxos::tx_pool` it uses `::mempool` instead.

Reply:
Happy to change it to anything else. Any suggestions?

Reply:
> Happy to change it to anything else. Any suggestions?

tx_mempool would be an ugly compromise. But personally, I would call it mempool and not worry about shadowing, because it seems like the global should be removed or renamed to `g_mempool` anyway, and doing that should be pretty easy, especially after these changes.

Reply:
Ok, left the name as is with to effects:

* Smaller diff
* `mempool` shadows `::mempool` until the global is renamed to `::g_mempoool`